protocol.md

```diff
@@ -236,6 +237,12 @@ Fired when the score changes.
 
 Contains only the `performance` property of [Status object](#status-object).
 
+### `energyChanged` event
+
+Fired when the energy (HP) changes.
```

Comment:
```suggestion
Fired when the energy (HP) changes. Energy is updated by the game even if the No Fail mod is enabled; it's up to the client to properly handle and display energy when that is the case.
```

Reply:
The energyChange event was in the code at one point, but I ended up removing it from emitting after the discussion on when energy changes.